app/models/text_component.rb

```diff
@@ -17,6 +17,8 @@ class TextComponent < ActiveRecord::Base
   delegate :name, to: :topic, prefix: true, allow_nil: true
   delegate :email, to: :assignee, prefix: true, allow_nil: true
 
+  enum publication_status: [ :draft, :fact_checked, :published ]
```

Comment:
if you have a default value in the migration, you should make this a Hash and define the value `0`